Module Federation

MonPote opened the pull request.

bert-e commented:
# Hello monpote,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/zenko-ui/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZKUI-371 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.1.0`


Please check the `Fix Version/s` of ZKUI-371, or the target
branch of this pull request.

MonPote committed:
ZKUI-371 Refactoring to introduce module federation

ZKUI-371 - update webpack configs

ZKUI-371 - update webpack's deps

ZKUI-371 - fix build

fix svg warning

ZKUI-371 - fix security check

fix dev webpack environnement

fix monaco issue

replace push from connected-react-router for history in react component

remove navbar

fix user creation

fix grafana url

fix all tests

update core-ui

fix rebase

Remove legacy cypress tests

fix linter errors

make prod ready

change public path

fix sts endpoint

fix

debug

fix

change client endpoint

Solve s3 proxy issues for object with keys containing urlencoded chars

This commit is the integration of #554

fix s3client

add basePath in s3client

Fix security workflow

fix test

Add go template

fix ci

Remove index.html

remove default to index.html

fix basepath for zenko client

MonPote commented:
ping

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @monpote

* @jbwatenbergscality

* @chengyanjin

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @jbwatenbergscality

* @monpote

* @chengyanjin

MonPote commented:
/approve

JBWatenbergScality reviewed (approved):
LGTM !

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/zenko-ui/commit/25963f33cefaeb92fe401ec291748eb8b2e66009) did not succeed in branch improvement/ZKUI-371-upgrade-deps-for-to-prepare-module-federation.




*The following options are set:* **approve**

JBWatenbergScality commented:
/status

bert-e commented:
# Status





*Status report is not available.*






*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/zenko-ui/commit/25963f33cefaeb92fe401ec291748eb8b2e66009) did not succeed in branch improvement/ZKUI-371-upgrade-deps-for-to-prepare-module-federation.




*The following options are set:* **approve**

MonPote committed:
fix ci

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/zenko-ui/commit/01490d0c582669e7e3d71af57effd87395e495b8) did not succeed in branch improvement/ZKUI-371-upgrade-deps-for-to-prepare-module-federation.




*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.1`



The following branches will **NOT** be impacted:

* `development/1.4`
* `development/1.5`
* `development/1.6`
* `development/2.0`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.1`



The following branches have **NOT** changed:

* `development/1.4`
* `development/1.5`
* `development/1.6`
* `development/2.0`



Please check the status of the associated issue ZKUI-371.

Goodbye monpote.
